develop/dev-guide-proxysql-integration.md

```diff
@@ -1,682 +1,1115 @@
+    <SimpleTab groupId="os">
 
-    mysql -u root -h 127.0.0.1 -P 4002 << EOF
-    DROP TABLE IF EXISTS test.test;
-    CREATE TABLE test.test (db VARCHAR(255));
-    INSERT INTO test.test (db) VALUES ('tidb-1');
-    EOF
+    <div label="macOS" value="macOS">
 
-    mysql -u root -h 127.0.0.1 -P 4003 << EOF
-    DROP TABLE IF EXISTS test.test;
-    CREATE TABLE test.test (db VARCHAR(255));
-    INSERT INTO test.test (db) VALUES ('tidb-2');
-    EOF
+    ```bash
+    docker compose down
+    cd -
+    ```
+
+    </div>
+
+    <div label="CentOS" value="CentOS">
+
+    ```bash
+    docker compose down
+    cd -
     ```
 
-3. To execute the [`proxysql-prepare.sql`](https://github.com/Icemap/tidb-proxysql-integration-test/blob/main/example/load-balance-admin-interface/proxysql-prepare.sql) in **_ProxySQL Admin Interface_**, execute the `docker-compose exec` command as follows:
+    </div>
+
+    <div label="Windows (Git Bash)" value="Windows">
 
-    ```shell
-    docker-compose exec proxysql sh -c "mysql -uadmin -padmin -h127.0.0.1 -P6032 < ./proxysql-prepare.sql"
+    ```bash
+    docker compose down
+    cd -
     ```
 
-    The preceding SQL file runs and triggers the following operations:
+    </div>
+
+    </SimpleTab>
+
+## Production environment
+
+For a production environment, it is recommended that you use [TiDB Cloud](https://en.pingcap.com/tidb-cloud/) directly for a fully managed experience.
+
+### Prerequisite
 
-    1. Adds hosts of three TiDB Servers and set all `hostgroup_id` as `0`.
-    2. Makes the configuration of TiDb Servers effective and saves it on disk.
-    3. Adds a `root` user with an empty password and sets `default_hostgroup` as `0`, corresponding to the preceding `hostgroup_id` of TiDB Servers.
-    4. Makes the configuration of the user effective and saves it on disk.
+- Download and install a MySQL client. For example, [MySQL Shell](https://dev.mysql.com/downloads/shell/).
 
-4. Log in to **_ProxySQL MySQL Interface_** with the `root` user and query 5 times using the following statements. The expected output contains `'tidb-0'`, `'tidb-1'`, and `'tidb-2'` three different values.
+### Integrate TiDB Cluster with ProxySQL on CentOS
 
-    ```shell
-    mysql -u root -h 127.0.0.1 -P 6034 -t << EOF
-    SELECT * FROM test.test;
-    SELECT * FROM test.test;
-    SELECT * FROM test.test;
-    SELECT * FROM test.test;
-    SELECT * FROM test.test;
+ProxySQL can be installed on many different platforms. The document takes CentOS as an example.
+
+For a full list of supported platforms and the corresponding version requirements, see [ProxySQL documentation](https://proxysql.com/documentation/installing-proxysql/).
+
+#### Step 1. Create a TiDB Cloud Dedicated Tier cluster
+
+For detailed steps, see [Create a TiDB Cluster](https://docs.pingcap.com/tidbcloud/create-tidb-cluster).
+
+#### Step 2. Install ProxySQL
+
+1. Add ProxySQL to the YUM repository:
+
+    ```bash
+    cat > /etc/yum.repos.d/proxysql.repo << EOF
```

Comment:
```suggestion
    cat > /etc/yum.repos.d/proxysql.repo << EOF
    
```

Reply:
Why need a blank line in the config writing command?